Orange/widgets/data/oweditdomain.py

```diff
@@ -662,6 +673,168 @@ def on_label_selection_changed(self):
         self.remove_label_action.setEnabled(bool(len(selected)))
 
 
+class GroupLessFrequentItemsDialog(QDialog):
+    """
+    A dialog for group less frequent values.
+    """
+    def __init__(
+            self, variable: Categorical, data: Union[np.ndarray, List],
+            selected_attributes: List[str], parent: QWidget = None,
+            flags: Qt.WindowFlags = Qt.Dialog, **kwargs
+    ) -> None:
+        super().__init__(parent, flags, **kwargs)
+        self.variable = variable
+        self.data = data
+        self.selected_attributes = selected_attributes
+
+        # grouping strategy
+        self.selected_radio = radio1 = QRadioButton("Group selected values")
+        self.frequent_abs_radio = radio2 = QRadioButton(
+            "Group values with less than"
+        )
+        self.frequent_rel_radio = radio3 = QRadioButton(
+            "Group values with less than"
+        )
+        self.n_values_radio = radio4 = QRadioButton(
+            "Group all except"
+        )
+
+        # if selected attributes available check the first radio button,
+        # it is most probable option
+        if selected_attributes:
+            radio1.setChecked(True)
+        else:
+            radio2.setChecked(True)
+
+        label2 = QLabel("occurrences")
+        label3 = QLabel("occurrences")
+        label4 = QLabel("the most frequent values")
+
+        self.frequent_abs_spin = spin2 = QSpinBox()
+        max_val = len(data)
+        spin2.setMinimum(1)
+        spin2.setMaximum(max_val)
+        spin2.setValue(10)
+        spin2.setMinimumWidth(
+            self.fontMetrics().width("X") * (len(str(max_val)) + 1) + 20
+        )
+        spin2.valueChanged.connect(partial(self._spin_changed, spin2))
+
+        self.frequent_rel_spin = spin3 = QDoubleSpinBox()
+        spin3.setMinimum(0)
+        spin3.setDecimals(1)
+        spin3.setSingleStep(0.1)
+        spin3.setMaximum(100)
+        spin3.setValue(10)
+        spin3.setMinimumWidth(self.fontMetrics().width("X") * (2 + 1) + 20)
+        spin3.setSuffix(" %")
+        spin3.valueChanged.connect(partial(self._spin_changed, spin3))
+
+        self.n_values_spin = spin4 = QSpinBox()
+        max_val = min(10, len(variable.categories))
```

Comment:
Is this limit needed? Couldn't we just limit to `len(variable.categories)`?

Reply:
Actually limit was `len(variable.categories)`, it was just a setting for the current number. The name of the variable was misleading.

Reply:
Anyway, it is changed now.